examples/Training/java/src/training/HowToUseTables.java

```diff
@@ -114,6 +132,23 @@ private void createTable()
             table.addData(newRow);
             OriginalFile file = table.getOriginalFile(); // if you need to interact with the table
             file = new OriginalFileI(file.getId(), false);
+
+            DataManagerFacility dm = gateway.getFacility(DataManagerFacility.class);
+
+            FileAnnotation annotation = new FileAnnotationI();
+            annotation.setFile(file);
+            annotation.setNs(omero.rtypes
+                    .rstring(omero.constants.namespaces.NSBULKANNOTATIONS.value));
+            annotation = (FileAnnotation) dm.saveAndReturnObject(ctx, annotation);
```

Comment:
It also means a possibly unnecessary round-trip to the server which decreases performance on both sides if this is done in bulk: a tricky decision for training materials!